be/src/cloud/cloud_rowset_writer.cpp

```diff
@@ -34,7 +35,14 @@ Status CloudRowsetWriter::init(const RowsetWriterContext& rowset_writer_context)
     if (_context.is_local_rowset()) {
         // In cloud mode, this branch implies it is an intermediate rowset for external merge sort,
         // we use `global_local_filesystem` to write data to `tmp_file_dir`(see `local_segment_path`).
-        _context.tablet_path = io::FileCacheFactory::instance()->get_cache_path();
+        if (auto maybe_cache_path = io::FileCacheFactory::instance()->get_cache_path();
```

Comment:
so cache path will be used even if enable_file_cache = false ? This will increase user's cognitive load somehow. A new path should be used in my opinion.

FYI, with more Cache Storage being added to Doris, disk file system is not the only place for cached data, e.g. system RAM for Memory-based Cache Storage. In this case, cache path will have value but not a directory.

Reply:
We enable file cache by false after duscussion. As for the availability of cache path, currently every cache has a working disk path. If mem cache is introduced, we should modify the implementation of FileCacheFactory then.